src/OperationResponse.php

```diff
@@ -241,9 +299,9 @@ public function getResult()
      * @return Status|null The status of the operation in case of failure, or null if
      *                                 operationFailed() is false.
      */
-    public function getError()
+    public function getError(Message $lastProtoResponse = null)
```

Comment:
Where is this parameter used in this function? Doesn't `$this->lastProtoResponse` refer to the member field of the object, not the parameter?

Reply:
Good catch! This was refactored and so the parameter here should be removed.